### DISCUSSION TITLE
enable / disable a Stateful interactable according to fov position

### DISCUSSION THREAD

**Discussion by phwyfve** [microsoft/MixedRealityToolkit-Unity#11188] [Q&A] [answered]
Hi all,
I see that this kind of feature already exists for the rendering of mrtk UI interactables : the more in center of fov they are, the more brighter they appear. Now I wonder if some components also exist for modifying the interactivity of such elements ? (ie when the button is in a certain gaze area, is it automatically enabled), or what component should be used to access the properties used by the rendering part to develop such a feature?
Thanks

**Comment by Zee2** [comment, marked as answer]
Hi @phwyfve, you can simply set the `.enabled` property of the interactable.
